src/UnityExtension/Assets/Editor/GitHub.Unity/UI/SettingsView.cs

```diff
@@ -94,6 +94,9 @@ public override void OnEnable()
         {
             base.OnEnable();
             AttachHandlers(Repository);
+
+            activeRemote = Repository != null ? Repository.CurrentRemote : null;
+            remoteHasChanged = true;
```

Comment:
Yeah, you're right, there's no point in having the field if it's only ever touched here, and it shouldn't be touched anywhere else anyway